### PR TITLE
Update: RRN ingestion

### PR DESCRIPTION
**Pull request by jping0929**

### CODE REVIEW

**Comment by jping0929** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "rrn-ingestion",
    "component": "gaia",
    "pr-id": "841",
    "commit-sha": "81168ef04b0405e0cb9394d0aa1c8af2b7d6be47",
    "pipeline": "master"
  },
  {
    "project": "rrn-ingestion",
    "component": "prismacloud",
    "pr-id": "93",
    "commit-sha": "bd5feba2f5da1f1487f9ee6cbadfeb9cc1662fbd",
    "pipeline": "master"
  },
  {
    "project": "rrn-ingestion",
    "component": "backend",
    "pr-id": "1895",
    "commit-sha": "28128bddb376e61d9193e61fe32102e8b696cef4",
    "pipeline": "master"
  }
]
```

File: specs/@prismabase.abs [hunk 1]
```diff
@@ -20,6 +20,15 @@ attributes:
     setter: true
     omit_empty: true
 
+  - name: RRN
```
Comment (t00f) [inline specs/@prismabase.abs:23]: @abhijitherekar @sawyerawesome is that ok to have the `RRN` property on `prismabase` ? If yes, this looks good to me

**Comment by jping0929** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "rrn-ingestion",
    "component": "backend",
    "pr-id": "1895",
    "commit-sha": "24756f905ad8ed62c2a4c0be06e4a05a144fd16e",
    "pipeline": "master"
  },
  {
    "project": "rrn-ingestion",
    "component": "gaia",
    "pr-id": "841",
    "commit-sha": "ede0809be5648a479d94ce5d7b89c60a883ccb44",
    "pipeline": "master"
  },
  {
    "project": "rrn-ingestion",
    "component": "prismacloud",
    "pr-id": "93",
    "commit-sha": "bd5feba2f5da1f1487f9ee6cbadfeb9cc1662fbd",
    "pipeline": "master"
  }
]
```

File: specs/@prismabase.abs (hunk 1 above)
Comment (abhijitherekar) [reply specs/@prismabase.abs:23]: yes, that should be ok.

**Review by abhijitherekar — APPROVED** [review]

**Pull request merged**